libs/api/domains/education/src/lib/graphql/frigg/organization.model.ts

```diff
@@ -43,4 +44,7 @@ export class FriggOrganizationModel {
 
   @Field({ nullable: true })
   website?: string
+
+  @Field({ nullable: true })
+  address?: AddressModel
```

Comment:
Shouldn't the type be put in `@Field` as well?
```suggestion
  @Field(() => AddressModel, { nullable: true })
  address?: AddressModel
```